frontend/app/mappers/benefit-application-service-mappers.server.ts

```diff
@@ -49,7 +49,7 @@ export function toBenefitApplicationRequest({
           PrivateDentalInsuranceIndicator: dentalInsurance,
           DisabilityTaxCreditIndicator: disabilityTaxCredit,
           LivingIndependentlyIndicator: livingIndependently,
-          InsurancePlan: toInsurancePlan(dentalBenefits),
+          InsurancePlan: dentalBenefits ? toInsurancePlan(dentalBenefits) : undefined,
```

Comment:
Uses Logical AND (&&) operator

```suggestion
          InsurancePlan: dentalBenefits && toInsurancePlan(dentalBenefits),
```

Reply:
Changed to &&